AIMAID-7-migration-table-people

ArthurSSR-alt opened the pull request:
Migration for table people where ID is auto increment and PK, Name is a string type with limit of 100 characters, NIF is a string type with limit of 9 characters and is unique (can't repeat), Birthdate is a date type, and timestamps for default predefinition of laravel.

ArthurSSR-alt committed:
Migration for table people

Migration for table people where ID is auto increment and PK, Name is a string type with limit of 100 characters, NIF is a string type with limit of 9 characters and is unique (can't repeat), Birthdate is a date type, and timestamps for default predefinition of laravel.

Pris-c reviewed (changes requested):
Rename the pull request title to match the branch name.
Remove the comments in Portuguese on each line.
Remove migrations that will not be used.

ArthurSSR-alt committed:
AIMAID-7-migration-table-people

Migration for table people where ID is auto increment and PK, Name is a string type with limit of 100 characters, NIF is a string type with limit of 9 characters and is unique (can't repeat), Birthdate is a date type, and timestamps for default predefinition of laravel. Now without the comments in portuguese.

ArthurSSR-alt committed:
Changes made in file migrations where table people is created
